### PR TITLE
Add missing #!/bin/bash

### PR DESCRIPTION
**Pull request by jcpunk**
A handful of platforms don't like the missing `#!/bin/bash` when running the test script.

### CODE REVIEW

File: tests.sh [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#/bin/bash -x
```
Comment (sbarzowski) [inline tests.sh:1]: Good point, but I wouldn't use `-x` here. This script already displays messages about its status.

Comment (jcpunk) [reply tests.sh]: Pushed new version without -x

**Review by sbarzowski — APPROVED** [review]

**Comment by sbarzowski** [comment]
Also while we're at it, I've found out that the proper way to do it is through `/usr/bin/env bash` as some more exotic system (e.g. NixOS, some BSDs) may not have `/bin/bash`.

**Comment by jcpunk** [comment]
Hello,

I've reached out to my supervisor on getting the CLA signed.  It might take a bit....

**Comment by jcpunk** [comment]
@googlebot I signed it!

**Comment by jcpunk** [comment]
@googlebot I signed it!

**Comment by jcpunk** [comment]
:) I think I'm all sorted now.

**Comment by sbarzowski** [comment]
Awesome, thanks.

**Pull request merged**